shoop/admin/templates/shoop/admin/products/_edit_media_form.jinja

```diff
@@ -0,0 +1,97 @@
+                <div class="col-sm-6 file-control">
+                    <div class="form-group{% if f.file.errors %} has-error{% endif %}">
+                        <span class="thumbnail">
+                            {% if f.thumbnail %}
+                                <a href="{{ f.file_url }}" target="_blank"><img src="{{ f.thumbnail }}"></a>
+                            {% else %}
+                                <small class="no-image">{% trans %}No image{% endtrans %}</small>
+                            {% endif %}
+                        </span>
+                        {{ f.file }}
+                        {% if f.file.errors %}
+                        <div class="help-block">{{ f.file.errors }}</div>
+                        {% endif %}
+                        {% if not is_image_form %}
+                            {{ bs3.field(f.external_url) }}
+                        {% endif %}
+                    </div>
+                </div>
+                <div class="col-sm-6 actions">
+                    {% if media_form.can_delete and f.instance.pk %}
+                    <a href="#" class="text-muted product-media-delete"><i class="fa fa-times"></i> {% trans %}Delete{% endtrans %}</a>
+                    <div class="hide">{{ f.DELETE }}</div>
+                    {% endif %}
+                    {{ bs3.field(f.ordering) }}
+                </div>
+            </div>
+        </div>
+        {%- set id = media_form.prefix + "-" + idx|safe %}
+        <div class="panel-footer" id="media-accordion-{{ id }}" role="tablist" aria-multiselectable="true">
+            <div role="tab" id="media-{{ id }}">
+                <div id="collapse-{{ id }}" class="advanced-container collapse" role="tabpanel" aria-labelledby="media-{{ id }}">
+                    <div class="advanced-body">
+                        {{ bs3.field(f.public) }}
+                        {{ bs3.field(f.purchased) }}
+                        {{ bs3.field(f.shops, widget_class="multiselect") }}
+                        {{ bs3.field(f.kind) }}
+                        {% call(form, language, map) language_dependent_content_tabs(f, tab_id_prefix="additional-language") %}
+                            {{ bs3.field(f[map.title]) }}
+                            {{ bs3.field(f[map.description]) }}
+                        {% endcall %}
+                    </div>
+                </div>
+            </div>
+            <div class="toggle-advanced">
+                <a role="button" data-toggle="collapse" data-parent="#media-accordion-{{ id }}" href="#collapse-{{ id }}" aria-expanded="false" aria-controls="collapse-{{ id }}">
+                    <i class="fa fa-cog"></i> {% trans %}Advanced options{% endtrans %}
+                </a>
+            </div>
+        </div>
+    </div>
+{% endmacro %}
+
+
+{%- set is_image_form = media_form.prefix == "images" %}
+{%- set name = "Product Images" if is_image_form else "Product Media" %}
+
+{% call content_block(_(name), "fa-image") %}
```

Comment:
Do the translation in the `{% set %}` statement; this won't be picked up by the i18n stack.